packages/blockchain-wallet-v4-frontend/src/data/components/interest/types.ts

```diff
@@ -18,16 +18,20 @@ export type InterestFormValuesType = {
 }
 
 export enum InterestSteps {
+  'DETAILS',
   'DEPOSIT'
```

Comment:
Should details come after DEPOSIT?

Reply:
enum evaluates to something like:

steps = {
  'DETAILS': 0,
  'DEPOSIT': 1
}

Then later we can use the indices to determine if the modal should go to the left or the right direction.

Reply:
Yes **DETAILS** should be the beginning because once user has deposited then details will be the first flyout they can see. I set up reducer to have **DEPOSIT** as default to deal with this edge case